src/index.js

```diff
@@ -23,11 +27,36 @@ app.engine(
     extname: "hbs",
     defaultLayout: "main",
     partialsDir: `${__dirname}/views/partials/`,
-    layoutsDir: `${__dirname}/views/layouts/`
+    layoutsDir: `${__dirname}/views/layouts/`,
+    helpers: {
+      __: function() { return i18n.__.apply(this, arguments); },
+      __n: function() { return i18n.__n.apply(this, arguments); }
```

Comment:
As far as I understand from the docs, we will probably not need ```__n``` anytime soon. :)

Reply:
Yes, but let's keep it anyways. Maybe we will need in future.